modules/client-common/src/main/java/org/apache/ignite/client/proto/ClientMessageDecoder.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.proto;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.ByteToMessageDecoder;
+import io.netty.util.CharsetUtil;
+import org.apache.ignite.lang.IgniteException;
+
+import java.nio.ByteBuffer;
```

Comment:
Import should be ordered alphabetically, but packages starting with 'java.' should be declared prior to others.

Reply:
Sorry but why? Who cares about import order? Does anyone even look at them? Modern IDEs add and remove imports automatically. I'd prefer not to waste time changing or reviewing their order.

Reply:
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-LineBreaking

Reply:
The document is 3 years old, we can change things for ignite-3. Import order does not affect anything - readability, maintainability, performance, etc. Right now we are wasting time on bikeshedding.

Reply:
Pavel, until we  don't change the rule we should follow them.

Reply:
Fixed.